idp-core/src/main/java/io/mosip/idp/core/util/Constants.java

```diff
@@ -9,15 +9,19 @@ public class Constants {
 
     public static final String CLIENT_ACTIVE_STATUS = "ACTIVE";
     public static final String UTC_DATETIME_PATTERN = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+    public static final String UTC_DATETIME_PATTERN_WITH_NANO_SECS = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSS.nnnnnnnnn";
```

Comment:
DateTimeFormatter.ISO_LOCAL_DATE_TIME is datetime till nanoseconds is without 'Z',  But still 'nnnnnnnnn' is not required here. Using DateTimeFormatter.ISO_LOCAL_DATE_TIME instead of this constant.